homeassistant/components/xiaomi_miio/strings.select.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Not sure it is a breaking change since there is state mapping

Reply:
That is for displaying it in the frontend, if one has an automation that sets an option, this breaks now as the states have changed.